src/main/java/se/sundsvall/supportmanagement/service/NotificationService.java

```diff
@@ -62,24 +64,23 @@ public List<Notification> getNotificationsByOwnerId(final String municipalityId,
 	}
 
 	public List<Notification> getNotificationsByErrandId(final String municipalityId, final String namespace, final String errandId, final Sort sort) {
+		accessControlService.verifyExistingErrandAndAuthorization(namespace, municipalityId, errandId, R, RW);
 		return notificationRepository.findAllByNamespaceAndMunicipalityIdAndErrandEntityId(namespace, municipalityId, errandId, sort)
 			.stream()
 			.map(NotificationMapper::toNotification)
 			.toList();
 	}
 
-	public List<Notification> getNotifications(final String municipalityId, final String namespace, final String ownerId) {
-		return notificationRepository.findAllByNamespaceAndMunicipalityIdAndOwnerId(namespace, municipalityId, ownerId).stream().map(NotificationMapper::toNotification).toList();
+	public String createNotification(final String municipalityId, final String namespace, final String errandId, final Notification notification) {
+		final var errandEntity = accessControlService.getErrand(namespace, municipalityId, errandId, false, RW);
+		return createNotification(municipalityId, namespace, errandEntity, notification);
 	}
 
-	public String createNotification(final String municipalityId, final String namespace, final String errandId, final Notification notification) {
+	public String createNotification(final String municipalityId, final String namespace, final ErrandEntity errandEntity, final Notification notification) {
```

Comment:
Can municipiality and namespace be retrieved from errandEntity instead of being method parameters?